cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
This class should be under cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/, it is not a global utility class (that's the purpose of cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/).